docs/news.rst

```diff
@@ -2,6 +2,36 @@
 Release Notes
 =============
 
+2.2.2 (2020-10-XX)
+==================
+
+Highlights
+----------
+
+- This version addresses an issue with downloads being retried unsuccessfully after encountering certain types of errors.
+- A `create_snapshot <index.html#synapseclient.Synapse.create_snapshot>`__ function is added for making table and view snapshots.
```

Comment:
I just realize that the term "snapshot" deviates from "version".  I used `create_snapshot` because the rest call said "snapshot". Thoughts?

Reply:
The rest response uses the "snapshotVersionNumber" key, would **create_snapshot_version** be clearer?

Reply:
Well it's slightly confusing, because "snapshots" isn't a term that is used widely in Synapse.  When you upload a new file, you create a new version.  We are creating table versions here, but we are calling them snapshots.  I wanted to call them `create_table_version` but then that wouldn't encapsulate views, so I just stuck with `create_snapshot` (also to mimic the rest call).  Thoughts @kelshmo ?

Reply:
Perhaps just adding to the documentation saying that `create_snapshot` is essentially creating versions of table or view  ?

Reply:
I think this is an important point - I do think `create_snapshot_version` is more clarifying than `create_snapshot`.

Reply:
Renamed.